src/Cli/Commands/InitOptions.cs

```diff
@@ -103,6 +105,9 @@ public class InitOptions : Options
         [Option("graphql.disabled", Default = false, Required = false, HelpText = "Disables GraphQL endpoint for all entities.")]
         public bool GraphQLDisabled { get; }
 
+        [Option("rest.request-body-strict", Default = CliBool.True, Required = false, HelpText = "When set to true, allows extraneous fields in the request body for REST.")]
+        public CliBool RestRequestBodyStrict { get; }
```

Comment:
How is this different from an option like `rest.disabled` with `Default = false`?

Reply:
When not included in the init command:
1. rest.disabled will assume the default value of false
2. rest.request-body-strict will assume the default value of true

With rest.disabled, you don't have the ability to set it to false since that is a boolean option.
With rest.request-body-strict, you can specify the value as true/false as you like.

Being an enum instead of bool, frees us of the need to use negation in the name as well.